XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -300,20 +300,20 @@ public static List<Host> HostsToUpgradeOrUpdate(Pool pool)
             if (pool == null)
                 return result;
 
-            var master = Helpers.GetMaster(pool);
-            if (master == null)
+            var coordinator = Helpers.GetCoordinator(pool);
+            if (coordinator == null)
                 return result;
 
             if (pool.IsMasterUpgraded())

```

Comment:
Contains master

Reply:
Excluded as it's part of XenAPI.

Reply:
IsMasterUpgraded() is API extension again and can change.